### PR TITLE
Disable inlineProjections

### PR DESCRIPTION
**Pull request by pontusmelke**
This reverts commit e930992e455eb03458a2a3626dc27d42d72f63f5.

### CODE REVIEW

File: community/server-examples/data/dbms/auth [hunk 1]
```diff
@@ -0,0 +1 @@
+neo4j:SHA-256,C55E199570E820F6A4318B88F9012E6798A0E48BE4E1CE1C3531636E0CA5DA5E,8204FEB576C766C02BEDAFCCF67CD7C9:password_change_required
```
Comment (davidegrohmann) [inline community/server-examples/data/dbms/auth]: What's this?

File: community/server-examples/data/dbms/authorization [hunk 2]
```diff
@@ -0,0 +1 @@
+neo4j:ee7e411ba5c6b8fce24ab3b3a7591bb7:SHA-256,F49E729BD48BDDCE17FEEDE0E29864CEF080BB3E69FAEC0022B7CF0366631430,B3266A4F46E62CD4D31F69F53857DC4B:password_change_required
```
Comment (davidegrohmann) [inline community/server-examples/data/dbms/authorization]: What's this?

File: community/server-examples/neo4j-home/data/dbms/authorization [hunk 3]
```diff
@@ -0,0 +1 @@
+neo4j:SHA-256,A472448B983A1252DE6E299FA186A597DE0DE8CF27FEDF6FC77254F1615ED339,853620E44023D75668308D64C89D5A01:password_change_required
```
Comment (davidegrohmann) [inline community/server-examples/neo4j-home/data/dbms/authorization]: What's this?

**Comment by pontusmelke** [comment]
@davidegrohmann thanks for the catch, `git revert` didn't do what I expected it to do…

**Pull request merged**